server/form-pages/apply/move-on/pduRegion.ts

```diff
@@ -0,0 +1,46 @@
+import type { TaskListErrors } from '@approved-premises/ui'
+import { validPostcodeArea } from '../../../utils/formUtils'
+
+import TasklistPage from '../../tasklistPage'
+
+export default class PduRegion implements TasklistPage {
+  name = 'pdu-region'
+
+  title = 'Placement duration and move on'
+
+  question = 'Probation delivery unit (PDU) postcode region'
```

Comment:
Are we sure this is the postcode region of the PDU? I was under the impression this would be the region the person would be relocating to?